traits/tests/test_map.py

```diff
@@ -25,8 +26,6 @@ class Person(HasTraits):
 
         person = Person()
 
-        self.assertEqual(Undefined, person.married)
```

Comment:
This test is duplicated in `test_no_default`, so we don't lose anything by removing it here.